src/amo/components/SearchResults/index.js

```diff
@@ -1,72 +1,72 @@
+/* @flow */
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import { compose } from 'redux';
 
 import AddonsCard from 'amo/components/AddonsCard';
 import { INSTALL_SOURCE_SEARCH } from 'core/constants';
 import translate from 'core/i18n/translate';
 import { hasSearchFilters } from 'core/searchUtils';
+import type { I18nType } from 'core/types/i18n';
 
 
-class SearchResults extends React.Component {
-  static propTypes = {
-    count: PropTypes.number,
-    filters: PropTypes.object,
-    i18n: PropTypes.object.isRequired,
-    loading: PropTypes.bool,
-    results: PropTypes.arrayOf(PropTypes.object),
-  }
+type Props = {|
+  count: number,
+  filters: Object,
```

Comment:
I'd be happy to follow-up on that, I only converted this file to flow but did not mean to fix everything here. The PR is already big enough IMHO. I do agree that we should do better here, but can we open a new issue for Flow?

Reply:
I guess a follow-up is fine, although in the past we've tried to avoid that for flow issues, as sometimes the follow-up never gets done. But I'm sure you'll make sure it gets fixed. 😄